### PR TITLE
Fix testing.sh concurrency regexp

### PR DESCRIPTION
**Pull request by sjanota**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

### CODE REVIEW

File: installation/scripts/testing.sh [hunk 1]
```diff
@@ -11,7 +11,7 @@ function validateConcurrency() {
     exit 1
   fi
 
-  if ! [[ "$1" =~ '^[0-9]+$' ]]; then
+  if ! [[ "$1" =~ ^[0-9]+$ ]]; then
```
Comment (jakkab) [inline installation/scripts/testing.sh]: Excuse me for being a pain in the neck, but as far as I can tell, this regexp matches leading zeros.

Comment (sjanota) [reply installation/scripts/testing.sh]: Yes, it does... Gimme a sec. I'll teach him good manners.

**Pull request merged**